internal/ethapi/api.go

```diff
@@ -1930,6 +2089,28 @@ func (s *PublicTransactionPoolAPI) SendTransaction(ctx context.Context, args Sen
 	if err != nil {
 		return common.Hash{}, err
 	}
+
+	// Quorum
+	if signed.IsPrivate() && s.b.IsPrivacyMarkerTransactionCreationEnabled() {
+		pmt, err := createPrivacyMarkerTransaction(s.b, signed, &args.PrivateTxArgs)
```

Comment:
this block seems to be quite similar to the block on https://github.com/ConsenSys/quorum/pull/1234/files#diff-c426ecd2f7d247753b9ea8c1cc003f21fa412661c1f967d203d4edf8163da344R499, could we remove duplication?

Reply:
That link doesn't work for me - it just takes me to the head of the PR.
However, this code hasn't changed from the reviewed PR. I'd be reluctant to do any refactoring now as I believe you want to get the PR merged as soon as possible.

Reply:
`internal/ethapi/api.go` line 488